UC1/src/main/java/es/uclm/esi/isoft2/a04/Persistence/BookingDAO.java

```diff
@@ -61,8 +58,7 @@ public void readBooking(Booking booking)
 			throws InstantiationException, IllegalAccessException, ClassNotFoundException, SQLException {
 		Vector<Vector<Object>> query_result;
 		String sql = "SELECT Client FROM Booking WHERE TableId = " + booking.getTable().getID() + " AND Date = '"
-				+ BookingDAO.mysqlDateSDF.format(booking.getDate()) + "' AND Turn = '" + booking.getTurn().toString()
-				+ "';";
+				+ this.mysqlDateSDF.format(booking.getDate()) + "' AND Turn = '" + booking.getTurn().toString() + "';";
```

Comment:
Access to an static element is best done by using the name of the class, the ssame as the other accesses

Reply:
This error appears multiple times, check by comparing develop and this branch